internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -489,17 +494,20 @@ func (s *LoadBalancedWebService) convertNetworkLoadBalancer() (networkLoadBalanc
 	config := networkLoadBalancerConfig{
 		settings: &template.NetworkLoadBalancer{
 			PublicSubnetCIDRs: s.publicSubnetCIDRBlocks,
-			Listener: template.NetworkLoadBalancerListener{
-				Port:            aws.StringValue(port),
-				Protocol:        strings.ToUpper(aws.StringValue(protocol)),
-				TargetContainer: targetContainer,
-				TargetPort:      targetPort,
-				SSLPolicy:       nlbConfig.SSLPolicy,
-				Aliases:         aliases,
-				HealthCheck:     hc,
-				Stickiness:      nlbConfig.Stickiness,
+			Listener: []template.NetworkLoadBalancerListener{
+				{
+					Port:            aws.StringValue(port),
+					Protocol:        strings.ToUpper(aws.StringValue(protocol)),
+					TargetContainer: targetContainer,
+					TargetPort:      targetPort,
+					SSLPolicy:       nlbConfig.SSLPolicy,
+					Aliases:         aliases,
+					HealthCheck:     hc,
+					Stickiness:      nlbConfig.Stickiness,
+				},
 			},
-			MainContainerPort: s.manifest.MainContainerPort(),
+			MainContainerPort:   s.manifest.MainContainerPort(),
+			CertificateRequired: certRequired,
```

Comment:
nit: we can get rid of the temp variable
```suggestion
			CertificateRequired: strings.ToUpper(aws.StringValue(protocol)) == manifest.TLS,
```

Reply:
can we get rid of the temp variable? the conversation was marked as resolved but the change wasn't updated :)